test/integration/patients/view.js

```diff
@@ -250,7 +250,9 @@ context('view page', function() {
     cy
       .get('.list-page__filters')
       .contains('Another Group');
+  });
 
+  specify('clinician in only one group', function() {
```

Comment:
this looks like it should be specify `group filtering - new actions` or something like that.  and there needs to be _another_ specify with one group down above 282.